packages/core/config/webpack/rules.js

```diff
@@ -106,29 +107,34 @@ module.exports = function getRules(context) {
       use: ['@svgr/webpack'],
     },
     {
-      resource: {
-        test: /\.jsx?$/,
-        or: [
-          include,
-          (filepath) => (
-            // These specific node modules, which contain arrow functions that must be
-            // transpiled.
-            filepath.includes('node_modules') &&
-              (
-                filepath.includes('query-string') ||
-                filepath.includes('split-on-first') ||
-                filepath.includes('strict-uri-encode') ||
-                filepath.includes('abort-controller') ||
-                filepath.includes('event-target-shim')
-              )
-          ),
-        ],
-      },
+      // resource: {
```

Comment:
Removing this will probably break IE11. We might be able to get around this with a separate loader, but it's kind of annoying.